src/main/java/uk/gov/hmcts/reform/sscs/controller/CitizenController.java

```diff
@@ -176,4 +181,33 @@ public ResponseEntity<Void> logUserWithCase(
 
         return ResponseEntity.noContent().build();
     }
+
+    @GetMapping(value = "/port-of-entries", produces = APPLICATION_JSON_VALUE)
```

Comment:
I might be being pedantic, but should this be /ports-of-entry

Reply:
yeah will make that change